src/Control/HTTPApplication.php

```diff
@@ -141,4 +152,85 @@ public function execute(HTTPRequest $request, callable $callback, $flush = false
             $this->getKernel()->shutdown();
         }
     }
+
+    private function warnAboutDeprecatedSetups()
```

Comment:
+ docblock pls

Reply:
I think it's pretty self explanatory but it's also easy to add so I'll add it.

Reply:
Done